contracts/cairo/deploy.sh

```diff
@@ -16,17 +22,24 @@ load_env() {
 }
 load_env
 
-CLASS_HASH=`starkli declare --rpc $SN_RPC_URL target/dev/yab_Escrow.sierra.json 2>&1 | grep -A1 "Class hash" | sed '1d'`
-
-echo $CLASS_HASH
-echo $HERODOTUS_FACTS_REGISTRY
-echo $ETH_CONTRACT_ADDR
-echo $MM_ETHEREUM_WALLET
-echo $SN_WALLET_ADDR
-echo $NATIVE_TOKEN_ETH_STARKNET
-
-CONTRACT_ADDRESS=`starkli deploy --rpc $SN_RPC_URL $CLASS_HASH $HERODOTUS_FACTS_REGISTRY $ETH_CONTRACT_ADDR $MM_ETHEREUM_WALLET $SN_WALLET_ADDR $NATIVE_TOKEN_ETH_STARKNET 2>&1 | grep -A1 "Contract deployed" | sed '1d'`
-
-echo $CONTRACT_ADDRESS
-
-starkli invoke --rpc $SN_RPC_URL eth approve $CONTRACT_ADDRESS 1000000000 1000000000
+echo -e "${GREEN}\n=> [SN] Declare Escrow${COLOR_RESET}"
+ESCROW_CLASS_HASH=$(starkli declare --watch --rpc $SN_RPC_URL target/dev/yab_Escrow.contract_class.json)
+
+echo -e "- ${PURPLE}[SN] Escrow ClassHash: $ESCROW_CLASS_HASH${COLOR_RESET}"
+echo -e "- ${PURPLE}[SN] Herodotus Facts Registry: $HERODOTUS_FACTS_REGISTRY${COLOR_RESET}"
+echo -e "- ${PURPLE}[SN] Market Maker: $SN_WALLET_ADDR${COLOR_RESET}"
+echo -e "- ${PURPLE}[SN] Ethereum ContractAddress $NATIVE_TOKEN_ETH_STARKNET${COLOR_RESET}"
+echo -e "- ${PINK}[ETH] Ethereum ContractAddress: $ETH_CONTRACT_ADDR${COLOR_RESET}"
+echo -e "- ${PINK}[ETH] Market Maker: $MM_ETHEREUM_WALLET${COLOR_RESET}"
+
+echo -e "${GREEN}\n=> [SN] Deploy Escrow${COLOR_RESET}"
+ESCROW_CONTRACT_ADDRESS=$(starkli deploy --watch --rpc $SN_RPC_URL $ESCROW_CLASS_HASH \
+    $HERODOTUS_FACTS_REGISTRY \
+    $ETH_CONTRACT_ADDR \
+    $MM_ETHEREUM_WALLET \
+    $SN_WALLET_ADDR \
+    $NATIVE_TOKEN_ETH_STARKNET)
+echo -e "- ${PURPLE}[SN] Escrow ContractAddress: $ESCROW_CONTRACT_ADDRESS${COLOR_RESET}"
+
+echo -e "${GREEN}\n=> [SN] Approve${COLOR_RESET}"
+starkli invoke --rpc $SN_RPC_URL eth approve $ESCROW_CONTRACT_ADDRESS 1000000000 1000000000
```

Comment:
```suggestion
```
This is no longer needed. This was used for the demo without frontend but this is no longer the case.